services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/enhanced/dynamodb/TableSchemaTest.java

```diff
@@ -85,6 +87,14 @@ public void fromClass_constructsImmutableTableSchema() {
         assertThat(tableSchema).isInstanceOf(ImmutableTableSchema.class);
     }
 
+    @Test
+    public void fromClass_missingObjectConverter() {
```

Comment:
Nit: Should contain the effect, maybe `fromClass_missingObjectConverter_throwsIllegalStateException`